src/path.ml

```diff
@@ -26,3 +26,8 @@ let relative_all p xs = List.fold_left Filename.concat p xs
 let join x y = Filename.concat x y
 
 let withExt x ~ext = x ^ ext
+
+let split x =
+  x |> Js.String.split sep |> Array.to_list |> List.filter (fun p -> p != "")
```

Comment:
How is it possible to get an empty component here?

Reply:
When the path is absolute, first element will be empty:
```ocaml
Path.split "/path/to" 
(* ["", "path", "to"] *)
```
Also, I think it's possible to construct `/path//to` paths, so empty path could be in the middle too. Probably the module needs some revising.

Reply:
I see. I think it's better to do away with `split` completely then. It's semantics aren't that clear on non normalized paths. For example, I would expect `foo/./bar` would remove the redundant `.`

Reply:
In case it helps, here is how esy looks for a manifest file

https://github.com/esy/esy/blob/master/bin/ProjectConfig.re#L110